### ISSUE TITLE
No getting any meanigful results

### ISSUE THREAD

**Issue by mbcel** [lydproject/UnSuperPoint_Project#3]
Hi, thank you for publishing this project.

I am trying to train my own model by using your simple_train() function. However, until now I am not able to get any meanigful results.
I am doing an overfitting test training by training only on a single image. Below are the results that I get after a few thousand batch iterations. From left to right [input image 1, input image 2, score heat map (high means high score), points positions]. As you can see the score map does not seem to give any meanigful keypoint positions. The same also applies to the realtive point positions.

![grafik](https://user-images.githubusercontent.com/16177860/90387110-e1b18080-e085-11ea-9eb6-1b4ddbd7f7e6.png)

Did you use the simple train function in the same form as you uploaded? Did you get any meningful results with this? I also saw you set batch size to 1. This seems unusual in combination with BatchNorm.
